pkg/virt-controller/watch/drain/evacuation/evacuation_test.go

```diff
@@ -30,80 +28,56 @@ import (
 )
 
 var _ = Describe("Evacuation", func() {
-	var ctrl *gomock.Controller
-	var stop chan struct{}
 	var virtClient *kubecli.MockKubevirtClient
-	var fakeVirtClient *kubevirtfake.Clientset
-	var vmiSource *framework.FakeControllerSource
-	var vmiInformer cache.SharedIndexInformer
-	var nodeSource *framework.FakeControllerSource
-	var nodeInformer cache.SharedIndexInformer
-	var migrationInformer cache.SharedIndexInformer
-	var migrationSource *framework.FakeControllerSource
-	var podInformer cache.SharedIndexInformer
-	var podSource *framework.FakeControllerSource
 	var recorder *record.FakeRecorder
-	var mockQueue *testutils.MockWorkQueue[string]
-	var kubeClient *fake.Clientset
-	var migrationFeeder *testutils.MigrationFeeder[string]
-	var vmiFeeder *testutils.VirtualMachineFeeder[string]
 
 	var controller *EvacuationController
 
-	syncCaches := func(stop chan struct{}) {
-		go vmiInformer.Run(stop)
-		go migrationInformer.Run(stop)
-		go nodeInformer.Run(stop)
-		go podInformer.Run(stop)
-
-		Expect(cache.WaitForCacheSync(stop,
-			vmiInformer.HasSynced,
-			migrationInformer.HasSynced,
-			nodeInformer.HasSynced,
-			podInformer.HasSynced,
-		)).To(BeTrue())
-	}
-
 	addNode := func(node *k8sv1.Node) {
-		mockQueue.ExpectAdds(1)
-		nodeSource.Add(node)
-		mockQueue.Wait()
+		controller.nodeStore.Add(node)
+	}
+	enqueue := func(node *k8sv1.Node) {
```

Comment:
Can there be places where `addNode` is called but `enqueue` should be not? Otherwise why not merge these two functions?